### ISSUE TITLE
Restart a specific app whose name contains comma from the ecosystem.json file

### ISSUE THREAD

**Issue by ps-rakeshchintha** [Unitech/pm2#5026]
## What's going wrong?
There is no way to restart a specific app whose name contains comma from the ecosystem.json file. 

## How could we reproduce this issue?
my ecosystem.json file
```
{
  "apps": [
    {
      "name": "account-1-products,families,map,catalog",
      "script": "main.js",
      "cwd": "/home/admin/myapp/",
      "exec_mode": "cluster",
      "instances": 1,
      "cron_restart": "55 6 * * *"
    },
    {
      "name": "account-2-products,families,map,catalog",
      "script": "main.js",
      "cwd": "/home/admin/myapp/",
      "exec_mode": "cluster",
      "instances": 1,
      "cron_restart": "55 6 * * *"
    }
  ]
}
```
When I run `pm2 start ecosystem.json --only account-2-products,families,map,catalog`, The app is not getting restarted because pm2 tries to restart 4 separate apps `account-2-products`, `families`, `map`, `catalog`. Which is fair as per the documentation https://pm2.keymetrics.io/docs/usage/application-declaration/#act-on-a-specific-process.

Because `,` is acting as a delimiter for apps, either pm2 should not allow app name to include `,` or there should be a way(probably through an escape character like /) to allow `,` as part of the app name.

**Comment by Tony308** [comment]
Have you tried speech marks for the value of the only flag? 
`pm2 start ecosystem.json --only "account-2-products,families,map,catalog"`

**Comment by ps-rakeshchintha** [comment]
Yup. That doesn't work as well. It still tries to start 4 apps which do not exist.
